### PR TITLE
HTML: Test rendering of fieldset with a tall legend

### PR DESCRIPTION
**Pull request by zcorpan**
Fixes https://github.com/whatwg/html/issues/4014

### CODE REVIEW

**Comment by zcorpan** [comment]
I don't understand why "hello" is 176 px down, and not 180px, but it seems to match browsers...

File: html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-tall-ref.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+.fieldset {
+  position: absolute;
+  z-index: -1;
+  margin-top: calc((/* half legend height */ 160px / 2) - (/* half top border */ 20px / 2));
+  background: green;
+  height: 40px;
+  left: 0;
+  right: 0;
+}
+.hello { margin-top: 176px; margin-left: 20px; }
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-tall-ref.html]: Assuming that 1em == 16px here? But maybe that's ok?

**Review by mstensho — DISMISSED** [review]

File: html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-tall-ref.html (hunk 1 above)
Comment (mstensho) [reply html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-tall-ref.html]: I misunderstood here. It should be 180px, not 176px. I really think we need to figure out why.

**Comment by zcorpan** [comment]
@MatsPalmgren do you know why the content is pushed down by this particular amount?

**Comment by mstensho** [comment]
In the ref, the top margin of .hello collapses with the bottom margin of the p. So 16px goes away there. So my original comment about assuming that 1em == 16px was right after all. :)

**Comment by zcorpan** [comment]
Aah, yes. It should be 160px (the height of the legend). Thanks.

**Review by mstensho — APPROVED** [review]

**Pull request merged**